cli/src/cli.rs

```diff
@@ -351,6 +351,8 @@ pub enum CliCommand {
         fee_payer: SignerIndex,
         authorized: SignerIndex,
         new_authorized: Option<SignerIndex>,
+        derived_address_seed: Option<String>, // Seed of current authority's derived key
+        derived_address_program_id: Option<Pubkey>, // Owner of current authority's derived key
```

Comment:
Sorry, but I changed my mind about exposing this in the cli. Let's not, until such support is actually needed.

Reply:
I peeled this off into #25948. We can decide to ship or abandon that, but it'll be there for posterity.

Reply:
Oh, I just saw this comment 🤦‍♀️ Great, we're on the same page! Thanks